src/src/tx_main.cpp

```diff
@@ -435,7 +438,7 @@ void ICACHE_RAM_ATTR HandleUpdateParameter()
     {
       /*uint8_t newRate =*/incRFLinkRate();
     }
-    Serial.println(ExpressLRS_currAirRate->enum_rate);
+    Serial.println(ExpressLRS_currAirRate_Modparams->enum_rate);
```

Comment:
Suppress?

Reply:
I think it's ok? Printing debug on the TX side is doesn't interfere with anything